packages/expo-firebase-core/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- Rename `IFirebaseOptions` type to `FirebaseOptions`, since it is not an interface. ([#14342](https://github.com/expo/expo/pull/14342) by [@Simek](https://github.com/Simek))
```

Comment:
```suggestion
- Rename `IFirebaseOptions` to `FirebaseOptions` and convert it to a type. ([#14342](https://github.com/expo/expo/pull/14342) by [@Simek](https://github.com/Simek))
```

Reply:
hmm, it was a `type` before, the only difference is that is no longer wrapped with `Partial`:
* https://github.com/expo/expo/pull/14342/files#diff-3d6e54fb5b9a09bb5225cb97b40ed3ec93d8abbf9013eb383b3e780bfc43becdL3

Reply:
ow, huh, I wasn't aware of that
now things are starting to make sense..
well in that case, yeah please ignore that suggestion